tests/Feature/Api/TasksTest.php

```diff
@@ -175,6 +175,36 @@ public function testSorting()
         $this->assertArraySubset(['completed_at' => null], $firstRow);
     }
 
+    public function testSortByRequestName()
+    {
+        //$request = factory(ProcessRequest::class)->create();
+
+        factory(ProcessRequestToken::class, 5)->create([
+            'user_id' => $this->user->id,
+            'completed_at' => Carbon::now()
+        ]);
+
+        // Order by process_request_id
+        $route = route('api.' . $this->resource . '.index',[
+                'order_by' => 'process_request_id',
+                'order_direction' =>'asc'
+        ]);
+        $response = $this->apiCall('GET', $route);
+        $response->assertStatus(200);
+        $firstRow = $response->json('data')[0];
+        $this->assertEquals(1, $firstRow['process_request_id']);
```

Comment:
@danloa this fails for me since my IDs don't always start at 1. Maybe in the factory get the list of ids with `->pluck('process_request_id')->sort()->toArray();` then you can reference $process_request_ids[0] like that